client/src/features/dashboard/components/ProjectsDashboard.tsx

```diff
@@ -80,36 +80,36 @@ function ProjectAlert({ total }: ProjectAlertProps) {
           <strong>You do not have any projects yet.</strong>
         </h3>
         <p>
-          If you are here for your first time, we recommend you go to through
-          our{" "}
+          You are accessing the &quot;legacy&quot; version of Renku. If you are
+          here for your first time, we recommend you go to{" "}
           <ExternalLink
             role="text"
-            title="tutorial"
+            title="Renku 2.0"
             className="fw-bold"
-            url={Docs.READ_THE_DOCS_TUTORIALS_STARTING}
+            url={"/"}
           />
-          . You can also{" "}
+          , and follow the{" "}
```

Comment:
Is the `,` on purpose here? Please ignore if my basic English doesn't cover this :stuck_out_tongue: 

```suggestion
          />{" "}
          and follow the{" "}
```

![Screenshot_20250604_160852](https://github.com/user-attachments/assets/ed442d3b-d2dd-4a27-a58f-a8eaf78189d9)